packages/lodestar/README.md

```diff
@@ -32,13 +27,26 @@ Open resulting html file in favorite browser.
 ## Usage
 1. If you haven't `yarn install`
 2. `lerna run build`
-3. `packages/lodestar/./bin/lodestar --help`
+3. `packages/lodestar-cli/bin/lodestar --help`
 
 Note:
 
 Spec test cases are optional dependency which can be skipped by adding `--ignore-optional` when installing dependencies.
 You can always download spec test cases by running `yarn install --force`.
 
+### Development
+
+#### Connect two beacon nodes
+
+`cd packages/lodestar-cli`
+
+Terminal 1:
+`bin/lodestar dev -r` - It will run beacon node and 8 validators with all the default settings. State will be written to .tmp/state.ssz
+
+Terminal 2:
+`bin/lodestar dev --bootnodes "/ip4/127.0.0.1/tcp30306" --multiaddrs "/ip4/127.0.0.1/tcp/30307" -v 0 -r` - Will connect to bootnode (node 1 default multiaddrs) and won't start validators.
```

Comment:
`tcp/30306`

Reply:
That's default port, but you cannot start two nodes on same machine with same port

Reply:
sorry, pointing to typo in the first bootnode string